Tests/id_set.json

```diff
@@ -11420,6 +11420,42 @@
                     "MISP V2 Test"
                 ]
             }
+        }, 
+        {
+            "MaxMind GeoIP2": {
+                "name": "MaxMind GeoIP2", 
+                "fromversion": "3.5.0", 
+                "commands": [
+                    "ip"
+                ], 
+                "tests": [
+                    "MaxMind Test"
+                ]
+            }
+        }, 
+        {
```

Comment:
Please remove these redundancies and merge from master, it should not create duplicates